OC4IDS_Data_Feedback_Notebook.ipynb

```diff
@@ -20,7 +20,7 @@
       "version": "3.8.2"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    flist = df2['fields'].tolist() #convert fields column to list
</pre><p>Don't abbreviate variable names and don't repeat the type of the variable in its name, e.g. use <code>fields</code> instead of <code>flist</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>